modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -69,7 +69,9 @@ If multiple instances of the same application are deployed, they will fail becau
 * You must have at least two Mule runtime engines in a cluster, each one running on different physical (or virtual) machines.
 * Mule high availability (HA) requires a reliable network connection between servers to maintain synchronization between the nodes in the cluster.
 * You must keep the following ports open to set up a Mule cluster: port `5701` and port `54327`.
-* You need to enable the multicast IP address: `224.2.2.3`. Multicast is used to discover new cluster members.
+* In case you configure your cluster to use multicast, you need to enable the multicast IP address: `224.2.2.3`.
```

Comment:
"If you configure..." I would also put the last sentence back as a second sentence in the bullet.